lib/introspector.js

```diff
@@ -169,9 +169,11 @@ const parseRest = lines => {
 // Returns: <Object>, function signature
 //   title - <string>, short function description
 //   description - <string>, extended function description
-//   parameters - <string[]>, function parameters
-//   comments - <string[]>, comments about returned value,
-//       thrown errors, deprecation and usage
+//   parameters - <Object[]>, function
+//       parameters, { name, types, nonStandardTypes, comment, offset }
```

Comment:
Nit: maybe wrap the object in backticks here and in similar cases, so that it would be rendered better in markdown?